WalletWasabi/Logging/Logger.cs

```diff
@@ -137,10 +137,10 @@ public static void Log(LogLevel level, string message, int additionalEntrySepara
 				}
 
 				message = Guard.Correct(message);
-				var category = string.IsNullOrWhiteSpace(callerFilePath) ? "" : $"{EnvironmentHelpers.ExtractFileName(callerFilePath)}:{callerMemberName} ({callerLineNumber})";
+				var category = string.IsNullOrWhiteSpace(callerFilePath) ? "" : $"{EnvironmentHelpers.ExtractFileName(callerFilePath)}.{callerMemberName} ({callerLineNumber})";
 
 				var messageBuilder = new StringBuilder();
-				messageBuilder.Append($"{DateTime.UtcNow.ToLocalTime():yyyy-MM-dd HH:mm:ss.fff} [{Thread.CurrentThread.ManagedThreadId}] {level.ToString().ToUpperInvariant()}\t");
+				messageBuilder.Append($"{DateTime.UtcNow.ToLocalTime():yyyy-MM-dd HH:mm:ss.fff} [{Environment.CurrentManagedThreadId}] {level.ToString().ToUpperInvariant()}\t");
```

Comment:
This is suggested change by VS2022 and described here https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1840. This is only refactoring. No functional change.